### PR TITLE
fix: [ui] modify ui display of propertydialog

### PR DESCRIPTION
**Pull request by LiHua000**
Adjust widget position and font size

Log: as title

bug: https://pms.uniontech.com/bug-view-190027.html
    https://pms.uniontech.com/bug-view-191155.html

### CODE REVIEW

**Review by itsXuSt — DISMISSED** [review]

**Comment by LiHua000** [comment]
recheck

File: src/plugins/common/dfmplugin-dirshare/widget/sharecontrolwidget.cpp [hunk 1]
```diff
@@ -240,20 +240,18 @@ void ShareControlWidget::setupUserName()
 
 void ShareControlWidget::setupSharePassword()
 {
-    sharePasswordlineEditor = new QLineEdit(this);
-    sharePasswordlineEditor->setStyleSheet("QLineEdit{background-color:rgba(0,0,0,0)}");   // just clear the background
+    sharePassword = new QLabel(this);
```
Comment (GongHeng2017) [inline src/plugins/common/dfmplugin-dirshare/widget/sharecontrolwidget.cpp]: 这里直接替换成DLabel吧。

**Review by GongHeng2017 — COMMENTED** [review]
+1

**Review by max-lvs — APPROVED** [review]
+2

**Pull request merged**